release/models/ospf/openconfig-ospfv2-lsdb.yang

```diff
@@ -1212,6 +1218,13 @@ submodule openconfig-ospfv2-lsdb {
         "The mask of the network described by the Summary LSA
         represented as a CIDR mask.";
     }
+
+    leaf metric {
+      type oc-ospf-types:ospf-metric;
+      description
+        "The cost of utilising the link specified independent of TOS";
```

Comment:
Is this description consistent with where we have metric elsewhere in the OSPF model?

Reply:
The description is a cut and paste of the router LSA metric description.   I updated to acknowledge which container this metric is in to differentiate the description from the network LSA.